spec/services/csv_exporter_spec.rb

```diff
@@ -4,33 +4,40 @@
 
   # let(:responding_team)   { create :responding_team, name: 'Transport for London' }
   # let(:responder)         { create :responder, full_name: 'Joe Smith', responding_teams: [ responding_team] }
+  let(:late_team)   { create :responding_team, name: 'Transport for London' }
   let(:foi_case)          { create :closed_case, :fully_refused_exempt_s40,
```

Comment:
Look nicer if the `let` were all aligned! And the above existing commented out `let` - worth deleting them?